libs/core/src/components/pds-box/pds-box.tsx

```diff
@@ -80,6 +92,36 @@ export class PdsBox {
   */
   @Prop() justifyContent?: `start` | `center` | `end` | `space-between` | `space-around`;
 
+  /**
+   * Defines the outer spacing between the box and another element.
+   * @defaultValue none
+  */
+  @Prop() margin?: BoxTShirtSizeType;
+
+  /**
+   * Defines the box's outer top spacing.
+   * @defaultValue none
+  */
+  @Prop() marginTop?: BoxTShirtSizeType;
```

Comment:
can we follow the logical naming conventions here? e.g `margin-block-start`. This should apply to the remaining margin Props below.

Reply:
+1

Reply:
I'm absolutely not against this, but will the engineers understand what those mean?

Reply:
I feel like with all the work and talk around localization, they're going to have to. Just my two cents.

Reply:
Good point!